src/Methods/NegativOrPostive.java

```diff
@@ -0,0 +1,42 @@
+package Methods;
+
+public class NegativOrPostive {
+    public static void main(String[] args) {
+        int[] arr = {-1, -2, -3};
+        double result = deviseToNegative(arr);
+        System.out.println("Result: " + result);
+
+    }
+
+    static double deviseToNegative(int[] arr) {
+        int positiveResult = 0;
+        int negativeResult = 0;
+
+        for (int i = 0; i < arr.length; i++) {
+            if (arr[i] >= 0) {
```

Comment:
1.Zero is not a positive number
2.why  do we have two results in code execution??
No postive numbers found.The result is -1
Result: -6.0
3. correct the name NegativOrPostive